manuscript/draft.Rmd

```diff
@@ -506,7 +506,7 @@ Most models produced confidence intervals that were too narrow, indicating
 overconfident predictions (Figure 5C). The random forest-based SDM stack was the
 most overconfident model, with only `r numbers$rf_coverage_pct`% of observations
 falling inside its 95% confidence intervals. This stacked SDM's narrow
-predictive distribution caused it to have notably higher deviance (Figure 5B)
+predictive distribution caused notably greater deviance (Figure 5B)
```

Comment:
I prefer the original here, but if we're going to make the change I think "resulted in notably greater deviance" is better.

I think the right solution here is probably something like this sentence, which address the awkwardness the editor is trying to tackle would be, without making the meaning of the sentence more vague:

> This narrow predictive distribution resulted in the stacked SDM having notably higher deviance...